test/Test/Database/LSMTree/Monoidal.hs

```diff
@@ -1,35 +1,32 @@
 {-# LANGUAGE AllowAmbiguousTypes #-}
-{-# LANGUAGE ScopedTypeVariables #-}
-{-# LANGUAGE TypeApplications    #-}
+{-# OPTIONS_GHC -fno-warn-orphans #-}
 
 module Test.Database.LSMTree.Monoidal (tests) where
 
 import           Control.DeepSeq (NFData)
-import           Data.Proxy (Proxy (Proxy))
 import           Data.Word
 import           Database.LSMTree.Extras.Generators ()
-import           Database.LSMTree.Internal.RawBytes (RawBytes)
 import           Database.LSMTree.Monoidal
 import           Test.Tasty
 import           Test.Tasty.QuickCheck
 
+-- | An example /(orphan)/ instance used to facilitate the property tests.
+instance ResolveValue Word64 where
+  resolveValue = resolveDeserialised (+)
+
 tests :: TestTree
 tests = testGroup "Test.Database.LSMTree.Monoidal"
-    [ testGroup "Word64" (allProperties @Word64 False)
-      -- TODO: revisit totality (drop requirement or fix @SerialiseValue Word64@)
+    [ testGroup "Word64" (allProperties @Word64)
```

Comment:
Ah, I thought we would just use the existing instance for `ResolveValue (Sum Word64)` instead of defining an orphan instance.